README.md

```diff
@@ -1,11 +1,28 @@
-# Your GitHub Learning Lab Repository for Community Starter Kit
+# Team Roundup bot: A Probot App
 
-Welcome to **your** repository for your GitHub Learning Lab course. This repository will be used during the different activities that I will be guiding you through to learn about making your project friendlier for collaboration.
+DESCRIPTION PLACEHOLDER
```

Comment:
Mona is part of a growing team of developers. To help onboard new teammates, Mona has created a slick Probot app called Welcome. Welcome automatically generates friendly messages when new team members create their first issue and open or merge their first pull request.